models/reset_password/reset_password.go

```diff
@@ -114,15 +114,15 @@ func (u *ResetPasswordStruct) Initialize() {
 	u.database = interfaceDB
 }
 
-func (u *ResetPasswordStruct) Save() bool {
+func (u *ResetPasswordStruct) Save() error {
```

Comment:
__[golint]__
* [golint] exported method ResetPasswordStruct.Save should have comment or be unexported